sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/downsample_filter/pickup_based_voxel_grid_downsample_filter_node.hpp

```diff
@@ -17,17 +17,24 @@
 
 #include "autoware/pointcloud_preprocessor/filter.hpp"
 
-#include <Eigen/Core>
-
-#include <pcl_conversions/pcl_conversions.h>
 #include <sensor_msgs/msg/point_cloud2.h>
 
-#include <algorithm>
-#include <cmath>
 #include <vector>
 
 namespace autoware::pointcloud_preprocessor
 {
+
+struct VoxelSize
+{
+  float x;
+  float y;
+  float z;
+};
+
```

Comment:
Now that we have this nice type, we could replace `voxel_size_x_` etc. and have a member `voxel_size_` instead. (Of course still instantiated from three ros parameters)